latest-langs

```diff
@@ -56,6 +56,7 @@ constant %paths = (
     'Raku'         => 'github.com/rakudo/rakudo/releases/latest',
     'Rebol'        => 'en.wikipedia.org/wiki/Rebol',
     'Rexx'         => 'regina-rexx.sourceforge.io',
+    'Rockstar 2'   => 'github.com/RockstarLang/rockstar2/releases/latest',
```

Comment:
This ought to work when v1.0.0 drops. I don't think there are current alternatives and it'll probably be our last and only resort to obtain the data.